tests/tests.toml

```diff
@@ -487,6 +487,10 @@ asr = true
 filename = "../integration_tests/global_syms_01.py"
 asr = true
 
+[[test]]
+filename = "../integration_tests/test_list_const.py"
```

Comment:
I think we need not add "../integration_tests/test_list_const.py" as a reference test here. We have it as an integration test and it is good.

Reply:
I need some guidance here too. If we register something as an integration test, do we not add it here? There is no information in the docs about registering tests in the `tests.toml`. We might as well add it there.

Reply:
>If we register something as an integration test, do we not add it here?

Sometimes we do and sometimes not. It depends on the specific case. In this case we test the "../integration_tests/test_list_const.py" as an integration test since we want to know if the test compiles and runs perfectly and produces the expected result.

When you want to compile and run a test example and also want to see/check its ASR output, you add the test in integration test and also enable it here in `tests.toml`.